raiden/storage/sqlite.py

```diff
@@ -8,7 +8,7 @@
 from raiden.utils import get_system_spec, typing
 
 # The latest DB version
-RAIDEN_DB_VERSION = 16
+RAIDEN_DB_VERSION = 17
```

Comment:
please, don't forget to do the migrations

Reply:
Also add test for each migration (in our case only one)

Reply:
Both are no my to-do list in the PR description